packages/wonder-blocks-tooltip/util/is-obscured.test.js

```diff
@@ -123,9 +132,80 @@ describe("isObscured", () => {
 
             // Assert
             expect(result).toBeTruthy();
-            // Reset the document method to avoid side-effects.
+            done();
+        };
+
+        arrange(actAndAssert);
+    });
+
+    test("element is not obscured, but elementFromPoint returns parent or child, returns false", (done) => {
+        // Arrange
+        let parentRef = null;
+        let elementRef = null;
+        let childRef = null;
+        const arrange = (tryActAndAssert) => {
+            const nodes = (
+                <View>
+                    <View
+                        ref={(ref) => {
+                            parentRef = ref;
+                            tryActAndAssert();
+                        }}
+                    >
+                        <View
+                            ref={(ref) => {
+                                elementRef = ref;
+                                tryActAndAssert();
+                            }}
+                        >
+                            <View
+                                ref={(ref) => {
+                                    childRef = ref;
+                                    tryActAndAssert();
+                                }}
+                            >
+                                Child
+                            </View>
+                        </View>
+                    </View>
+                </View>
+            );
+            mount(nodes);
+        };
+
+        const actAndAssert = () => {
+            if (!elementRef || !parentRef || !childRef) {
+                // We're not ready to act or assert yet.
+                // We need all three refs to be in.
+                return;
+            }
```

Comment:
Wow.

Reply:
I wish Github would show more of the code.

Reply:
I think if you drag select a group of lines, you can comment on the group and it should show them all.

Reply:
Hmm, I am mistaken. :(